app/assets/javascripts/advanced_search.js

```diff
@@ -25,6 +25,7 @@
       });
       $(".js-calendar-full").datepicker();
       $.datepicker.setDefaults($.datepicker.regional[locale]);
+      $.datepicker.setDefaults({ dateFormat: "dd/mm/yy" });
```

Comment:
How about a test to make sure we don't run into this annoying bug again?

This is what I've come up with. Feel free to improve it! :wink:

```ruby
banner = create(:banner)
login_as(create(:administrator).user)

visit edit_admin_banner_path(banner)
fill_in "Post started at", with: "20/02/2002"
find_field("Post started at").click
within(".ui-datepicker") { click_link "22" }

expect(page).to have_field "Post started at", with: "22/02/2002"

```

Reply:
I added the same spec you mentioned...I haven't found a simpler way to test it.

Related with this spec:

When I created the PR I already commented in the notes that an _improve manage dates on admin section_ is still pending. But now doing the spec I don't know if we should apply any improvement in terms of data consistency.

Look at the lines: https://github.com/consul/consul/blob/master/app/views/admin/banners/_form.html.erb#L8
https://github.com/consul/consul/blob/master/app/views/admin/banners/_form.html.erb#L15

These lines apply a format depending on the locale of the administrator. What happens if we edit only one of the two dates of the banner? The date just selected will be seen with the format "dd/mm/yyyy" and the date that has not been modified will be seen with the format "yyyy-mm-dd".
It's true that once you save the changes, they will both be rendered again with the same format, but until you save it, it can be confusing.

![Captura de pantalla 2020-02-04 a las 18 52 58](https://user-images.githubusercontent.com/16189/73771946-94bc2b80-477f-11ea-84b7-8cc74f785f3e.png)

Reply:
> What happens if we edit only one of the two dates of the banner? The date just selected will be seen with the format "dd/mm/yyyy" and the date that has not been modified will be seen with the format "yyyy-mm-dd".

Yeah, I think we mentioned it when discussing this issue last week. That's pretty horrible, actually :sweat_smile:. We should definitely use the same format in both cases, although in my opinion fixing it isn't that urgent and we can leave it as it is now until we release version 1.1 :thinking:.

Just one more thing about this pull request: would you mind squashing both commits into one? :pray: Thanks a lot!